windows/src/global/inc/keymanengine.h

```diff
@@ -0,0 +1,254 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * Name:             KeymanEngine
+ * Description:      Main header file for windows engine
+ *
+*/
+/***************************************************************************/   // I4006   // I4169
```

Comment:
Not sure how this version history works for e.g. `// I4006` but I have brought them all over throughout this file.

Reply:
Yeah, these are legacy issue references. We have the legacy issues in a private repo, e.g. https://github.com/sillsdev/keyman-issues/issues, so you can search for the tag `I4006` which returns https://github.com/sillsdev/keyman-issues/issues/3997.